net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.inc

```diff
@@ -312,14 +337,24 @@ if [ -e /var/run/quagga/ospfd.pid ]; then
 	/bin/pkill -F /var/run/quagga/ospfd.pid
 	/bin/rm -f /var/run/quagga/ospfd.pid
 fi
+if [ -e /var/run/quagga/bgpd.pid ]; then
+	kill -9 `cat /var/run/quagga/bgpd.pid`
+	rm -f /var/run/quagga/bgpd.pid
+fi
 EOF;
 	$rc_file_start = <<<EOF
+/bin/mkdir -p {$quagga_config_base}
+/bin/mkdir -p /var/etc/quagga
 /bin/mkdir -p /var/run/quagga
 /bin/mkdir -p /var/log/quagga
 /bin/rm -f /var/run/quagga/zebra.pid
 /bin/rm -f /var/run/quagga/ospfd.pid
+/bin/rm -f /var/run/quagga/bgpd.pid
 
+/bin/chmod 777 {$quagga_config_base}
+/bin/chmod 777 /var/etc/quagga
```

Comment:
Is this really necessary? It doesn't seem to be. If it is, just use the first line, the second is redundant ( `$quagga_config_base` is defined as being `/var/etc/quagga` )